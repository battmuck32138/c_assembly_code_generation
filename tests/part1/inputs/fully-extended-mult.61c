int main () {
  return ((1 * 3) * 5) * (2 * (3 * 8));
}
