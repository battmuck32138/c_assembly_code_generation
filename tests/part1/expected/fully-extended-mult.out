
Exited with error code 720
